source/includes/wc-analytics/_reports.md

```diff
@@ -0,0 +1,731 @@
+# Analytics Reports #
+
+The reports API enables you to view analytics data on products, sales, revenue, orders, and more.
+
+## List all the reports endpoints ##
+
+This API helps you to list all the endpoints regarding reports.
+
+### HTTP request ###
+
+<div class="api-endpoint">
+	<div class="endpoint-data">
+		<i class="label label-get">GET</i>
+		<h6>/wp-json/wc-analytics/reports</h6>
+	</div>
+</div>
+
+> Example of how to list all the enpoints regarding WC Analytics reports:
+
+```shell
+curl https://example.com/wp-json/wc-analytics/reports \
+	-u consumer_key:consumer_secret
+```
+
+```javascript
+WooCommerce.get("reports")
```

Comment:
are we upgrading the SDKs as part of this PR, or is this something that automatically happens?

Reply:
I think we don't need to upgrade the SDKs, I've [added in the doc](https://github.com/woocommerce/woocommerce-rest-api-docs/pull/249/files#diff-b93423f105f35b0fd4a5ab30c755597f236587ae4e8b36a3c9507e658bd8cccdR173) that we need to use `wc-analytics` instead of `wc/v3` when specifying version in the SDK settings. The code examples were also added for setting for all languages, here's [the PHP one](https://github.com/woocommerce/woocommerce-rest-api-docs/pull/249/files#diff-b93423f105f35b0fd4a5ab30c755597f236587ae4e8b36a3c9507e658bd8cccdR207).

I tried locally using PHP, Python, and Node.js and they can get the results under `/wc-analytics` namespace.